packages/dd-trace/src/appsec/iast/index.js

```diff
@@ -14,17 +14,20 @@ const IAST_ENABLED_TAG_KEY = '_dd.iast.enabled'
 const requestStart = dc.channel('dd-trace:incomingHttpRequestStart')
 const requestClose = dc.channel('dd-trace:incomingHttpRequestEnd')
 
-function enable (config) {
+function enable (config, _tracer) {
```

Comment:
You can access the tracer in a global var, something like `global._ddtrace.tracer` if i'm not wrong. I'm not sure which approach is best but at least now you know :+1:

Reply:
I think that you can also require the tracer instance, it is exported here: https://github.com/DataDog/dd-trace-js/blob/7fb068aff5a877360e968523935811832ae7a75f/packages/dd-trace/index.js#L17

Reply:
whoops ok, nevermind then